### PR TITLE
Statements

### PR DESCRIPTION
**Pull request by CZDiego**

### CODE REVIEW

File: sparky_type.py [hunk 1]
```diff
@@ -9,8 +9,8 @@
 class SparkyType:
     def __init__(self):
         self.spark_type = ""
-        self.col = 0
```
Comment (Sandman1l) [inline sparky_type.py]: ` python
self.col = 0
        self.row = 0
        self.col = 0
`

Comment (VF1Gimure) [inline sparky_type.py:12]:
```python
 self.col = 0
self.row = 0
```

Please review this.

File: varTable.py [hunk 2]
```diff
@@ -13,8 +13,7 @@ class Var:
 
 	def __init__(self):
 		self.s_type    = SparkyType()
-		self.scope     = "Global"
```
Comment (VF1Gimure) [inline varTable.py:16]:
```python
self.scope = "World"
```
Sugerencia a cambio :v

File: yacc.py [hunk 3]
```diff
@@ -264,7 +287,7 @@ def p_let1(p):
 def p_let2(p):
     'let2   :'
     program.current_var.s_type = program.current_type
-    program.current_var.address = program.globalMemory.get_next_address(program.current_type.spark_type)
+    program.current_var.address = program.globalMemory.get_next_address(program.current_type.spark_type, 0, 0)
```
Comment (VF1Gimure) [inline yacc.py:290]:
```python
program.current_var.address = program.globalMemory.get_next_address(program.current_type)
```
easier to understand.

**Review by VF1Gimure — APPROVED** [review]
good job

**Pull request merged**